xcsv.cc

```diff
@@ -163,7 +163,8 @@ enum xcsv_token {
   XT_YYYYMMDD_TIME
 };
 
-#include "xcsv_tokens.gperf"       // for Perfect_Hash, xt_mapping
+//#include "xcsv_tokens.gperf"       // for Perfect_Hash, xt_mapping
+#include "xcsv_tokens.cc"
```

Comment:
Can we make xcsv_tokens a member variable instead of polluting the global name space?
Is there an advantage to using an include file here?